src/components/v5/frame/NavigationSidebar/partials/NavigationSidebarMainMenu/NavigationSidebarMainMenu.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC } from 'react';
+import { AnimatePresence, motion } from 'framer-motion';
+
+import { useTablet } from '~hooks';
+import { accordionAnimation } from '~constants/accordionAnimation';
+
+import NavigationSidebarButton from '../NavigationSidebarButton';
+import NavigationSidebarSecondLevel from '../NavigationSidebarSecondLevel';
+import NavigationSidebarThirdLevel from '../NavigationSidebarThirdLevel/NavigationSidebarThirdLevel';
```

Comment:
it should be import NavigationSidebarThirdLevel from '../NavigationSidebarThirdLevel'; like others navigations above